components/const.py

```diff
@@ -221,3 +221,9 @@
     "yam",
     "zucchini",
 ]
+BUY_TEXT = (
+    "Hello there. You are writing your job offer in our technical focused groups, which is "
+    "against our rules. To find a bot developer, please look at agencies dedicated towards "
+    "freelancers. An example of those would be https://fiverr.com, with which we are not "
+    "associated of course."
```

Comment:
```suggestion
    "freelancers. An example of those would be https://fiverr.com, which we are not "
    "associated with."
```

Reply:
Just in case: both "with which" and "which... with" are grammatically correct, as far as I know :)